consensus/progpow/consensus.go

```diff
@@ -30,25 +30,9 @@ import (
 var (
 	allowedFutureBlockTimeSeconds = int64(15) // Max seconds from current time allowed for blocks, before they're considered future blocks
 
-	ContextTimeFactor = big10
 	ZoneBlockReward   = big.NewInt(5e+18)
-	RegionBlockReward = new(big.Int).Mul(ZoneBlockReward, big3)
-	PrimeBlockReward  = new(big.Int).Mul(RegionBlockReward, big3)
-)
-
-// Some useful constants to avoid constant memory allocs for them.
-var (
-	expDiffPeriod = big.NewInt(100000)
-	big0          = big.NewInt(0)
-	big1          = big.NewInt(1)
-	big2          = big.NewInt(2)
-	big3          = big.NewInt(3)
-	big8          = big.NewInt(8)
-	big9          = big.NewInt(9)
-	big10         = big.NewInt(10)
-	big32         = big.NewInt(32)
-	bigMinus99    = big.NewInt(-99)
-	big2e256      = new(big.Int).Exp(big.NewInt(2), big.NewInt(256), big.NewInt(0)) // 2^256
+	RegionBlockReward = new(big.Int).Mul(ZoneBlockReward, common.Big3)
```

Comment:
this should not be used right? we should delete these reward fields, rewards are always calculated per block basis

Reply:
This is from the common checker PR, will address there